Spigot-Server-Patches/0582-Prevent-fire-from-spreading-to-invalid-locations.patch

```diff
@@ -0,0 +1,18 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Mariell Hoversholm <proximyst@proximyst.com>
+Date: Mon, 21 Sep 2020 20:40:10 +0200
+Subject: [PATCH] Prevent fire from spreading to invalid locations
+
+
+diff --git a/src/main/java/net/minecraft/server/BlockFire.java b/src/main/java/net/minecraft/server/BlockFire.java
+index e2dd27be6b523c6e6668923c1b1900cb9251f39f..fbb62242b9eaa8a492e50130d479484e9aa84179 100644
+--- a/src/main/java/net/minecraft/server/BlockFire.java
++++ b/src/main/java/net/minecraft/server/BlockFire.java
+@@ -197,6 +197,7 @@ public class BlockFire extends BlockFireAbstract {
+                                 }
+ 
+                                 blockposition_mutableblockposition.a((BaseBlockPosition) blockposition, l, j1, i1);
++                                if (blockposition_mutableblockposition.isInvalidYLocation()) continue; // Paper - fires may not spread to invalid Y locations
```

Comment:
Lets just modify the existing patch adding the line below this so we can avoid another patch file for this.